documents/webhook.md

```diff
@@ -0,0 +1,227 @@
+## Webhook
+
+### Create a Webhook
+```C#
+
+String accountId = "acc_GP4lfNA0iIMn5B";
+
+Dictionary<string, object> webhookRequest = new Dictionary<string, object>();
```

Comment:
Have we checked the syntax for this?

Reply:
Yes i had checked all the code snippet manually while call the api's